README.md

```diff
@@ -4,33 +4,301 @@ sktm - skt manager
+
+    sudo -u jenkins tee ~jenkins/.beaker_client/config >/dev/null <<EOF
+    HUB_URL = "https://beaker.example.com"
+    AUTH_METHOD = "krbv"
+    KRB_REALM = "EXAMPLE.COM"
+    EOF
+
+If your Beaker instance is accessed via HTTPS you might need to supply the CA
+certificate as well, by adding a line like this to the above configuration:
+
+    CA_CERT = "<CA_CERT>"
+
+Where `<CA_CERT>` would be a path to a CA certificate file in PEM format. E.g.
+
+    sudo -u jenkins tee ~jenkins/.beaker_client/config >/dev/null <<EOF
+    HUB_URL = "https://beaker.example.com"
+    AUTH_METHOD = "krbv"
+    KRB_REALM = "EXAMPLE.COM"
+    CA_CERT = "/etc/beaker/ca.pem"
+    EOF
+
+If you have a keytab for a dedicated principal to use for accessing Beaker,
+you can supply it by adding these two lines to the configuration file:
+
+    KRB_KEYTAB = "<KEYTAB>"
+    KRB_PRINCIPAL = "<JENKINS_PRINCIPAL>"
+
+Here `<KEYTAB>` would be a path to the keytab file, and `<JENKINS_PRINCIPAL>`
+would be the Kerberos principal for Jenkins to use. For example (together with
+the CA certificate):
+
+    sudo -u jenkins tee ~jenkins/.beaker_client/config >/dev/null <<EOF
+    HUB_URL = "https://beaker.example.com"
+    AUTH_METHOD = "krbv"
+    KRB_REALM = "EXAMPLE.COM"
+    CA_CERT = "/etc/beaker/ca.pem"
+    KRB_KEYTAB = "/etc/beaker/keytab"
+    KRB_PRINCIPAL = "jenkins/special-principals.example.com"
+    EOF
+
+Make sure to take steps to protect the keytab file in production environments.
+
+If you don't have a dedicated principal and a keytab, you can omit
+`KRB_KEYTAB` and `KRB_PRINCIPAL` settings, and let Jenkins use your Kerberos
+credentials like this:
+
+    sudo -u jenkins kinit <YOUR_PRINCIPAL>
+
+Where `<YOUR_PRINCIPAL>` would be the principal you usually login with. E.g.:
+
+    sudo -u jenkins kinit user@EXAMPLE.COM
+
+Note that in this case you would need to periodically refresh the credentials
+Jenkins has as they expire, by re-running the command above.
+
+Make sure you get a dedicated principal and a keytab in production
+environment, instead of letting Jenkins use your credentials.
+
+### Jenkins
+
+To configure Jenkins, open [http://localhost:8080](http://localhost:8080) in
+your browser and follow the wizard. Installing the suggested plugins should be
+sufficient. The examples below assume that the admin user, created during
+Jenkins configuration, was named "sktm", and assigned password "sesame".
+
+At the moment, sktm doesn't support CSRF protection when authenticating to
+Jenkins. So, after the Jenkins setup is complete, navigate to [Manage Jenkins
+-> Configure Global Security](http://localhost:8080/configureSecurity/) and
+uncheck the "Prevent Cross Site Request Forgery exploits" checkbox. Note that
+you shouldn't do this on production, or exposed systems.
+
+Create a configuration file for Jenkins Job Builder, describing how to access
+Jenkins:
+
+    mkdir -p ~/.config/jenkins_jobs
+    cat >~/.config/jenkins_jobs/jenkins_jobs.ini <<"EOF"
```

Comment:
Why are we quoting `EOF` here and nowhere else?

Reply:
Right, should be quoted everywhere. Fixing. Thanks!